src/Service.Tests/SqlTests/GraphQLMutationTests/MultipleCreateMutationTests/MultipleCreateMutationTestBase.cs

```diff
@@ -0,0 +1,819 @@
+
+        /// <summary>
+        /// Point multiple create mutation request is executed with the role: role_multiple_create_policy_tester. This role has the following create policy defined on "Book" entity: "@item.title ne 'Test'"
+        /// Because this mutation tries to create a book with title "Test", it is expected to fail with a database policy violation error. The error message and status code are validated for accuracy.
+        /// </summary>
+        public async Task PointMultipleCreateFailsDueToCreatePolicyViolationAtTopLevelEntity(string expectedErrorMessage, string bookDbQuery, string publisherDbQuery)
+        {
+            string graphQLMutationName = "createbook";
+            string graphQLMutation = @"mutation{
+                                          createbook(item:{
+                                            title: ""Test"",
+                                            publishers:{
+                                              name: ""Publisher #1""
+                                            }
+                                          }){
+                                            id
+                                            title
+                                            publishers{
+                                              id
+                                              name
+                                            }
+                                          }
+                                        }";
+
+            JsonElement actual = await ExecuteGraphQLRequestAsync(graphQLMutation, graphQLMutationName, isAuthenticated: true, clientRoleHeader: "role_multiple_create_policy_tester");
+
+            SqlTestHelper.TestForErrorInGraphQLResponse(
+                response: actual.ToString(),
+                message: expectedErrorMessage,
+                statusCode: $"{DataApiBuilderException.SubStatusCodes.DatabasePolicyFailure}"
+            );
+
+            // Validate that no book item is created
+            string dbResponse = await GetDatabaseResultAsync(bookDbQuery);
+            Assert.AreEqual("[]", dbResponse);
+
+            // Validate that no publisher item is created
+            dbResponse = await GetDatabaseResultAsync(publisherDbQuery);
+            Assert.AreEqual("[]", dbResponse);
+        }
+
+        /// <summary>
+        /// Point multiple create mutation request is executed with the role: role_multiple_create_policy_tester. This role has the following create policy defined on "Publisher" entity: "@item.name ne 'Test'"
+        /// Because this mutation tries to create a publisher with title "Test" (along with creating a book item), it is expected to fail with a database policy violation error.
+        /// As a result of this mutation, no Book and Publisher item should be created.  
+        /// The error message and status code are validated for accuracy. Also, the database is queried to ensure that no new record got created.
+        /// </summary>
```

Comment:
```suggestion
        /// Point multiple create mutation request is executed with the role: role_multiple_create_policy_tester. 
        /// This role has the following create policy defined on "Publisher" entity: "@item.name ne 'Test'"
        /// Because this mutation tries to create a publisher with title "Test" (along with creating a book item),
        /// it is expected to fail with a database policy violation error.
        /// As a result of this mutation, no Book and Publisher item should be created.  
        /// The error message and status code are validated for accuracy. 
        /// Also, the database is queried to ensure that no new record got created.
        /// </summary>
```